Leanplum/Cache/LPStorageSQLite.m

```diff
@@ -0,0 +1,234 @@
+//
+//  LPDatabase.m
+//  Leanplum
+//
+//  Created by Alexis Oyama on 6/9/17.
+//  Copyright (c) 2017 Leanplum, Inc. All rights reserved.
+//
+//  Licensed to the Apache Software Foundation (ASF) under one
+//  or more contributor license agreements.  See the NOTICE file
+//  distributed with this work for additional information
+//  regarding copyright ownership.  The ASF licenses this file
+//  to you under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+
+#import "LPStorageSQLite.h"
+#import "LPFileManager.h"
+#import "LPConstants.h"
+#import <sqlite3.h>
+
+static sqlite3 *sqlite;
```

Comment:
same here. please make this a local property

Reply:
same as above. We can definitely split this logic, but we will need to investigate how to use these shared values. May be a PR for this to split will help.